mediaqueries-5/Overview.bs

```diff
@@ -478,6 +478,36 @@ Detecting the desire for less motion on the page: the 'prefers-reduced-motion' f
 
 	</dl>
 
+<h3 id="prefers-reduced-transparency">
+Detecting the desire to reduce the use of transparent or translucent layering effects on the page: the 'prefers-reduced-transparency' feature</h3>
```

Comment:
Agree that the standard English usage of all of these related terms are all roughly equivalent, and all encompass degrees.

For example, I was taught that opaque/translucent/transparent refer to the degree of opacity: something is translucent when light passes thru, but it's not clear enough to actually see details on the far side; something is transparent when you can actually see thru it to at least some degree.  (The boundary is obviously very fuzzy, no pun intended, but the categories are clear enough, no pun intended.  "Privacy glass" is more on the translucent side, cloudy plastic is translucent, etc.  Normal windows are transparent.  Rice paper is translucent.)

In other words, the definitions are definitely intertwined and uncertain enough that we can really use any of them here; there's no way to defend any particular definition as Definitely Correct.